.github/workflows/release-github.yml

```diff
@@ -0,0 +1,22 @@
+name: Release on GitHub
+
+on:
+  push:
+    branches: [release/*]
+
+jobs:
+  pre-release-check:
+    uses: cucumber/.github/.github/workflows/prerelease-checks.yml@main
+
+  create-github-release:
+    name: Create GitHub Release and Git tag
```

Comment:
This uses the changelog tool. Does the current changelog format work with that?

Reply:
Good question. I'll test it.

Reply:
Yep, it [looks like this CHANGELOG trips up the tools' parser](https://github.com/cucumber/cucumber-jvm/runs/5588154935?check_suite_focus=true) at the moment.

Although this feature (creating a GitHub release) is nice, it's not essential. Why don't we remove it for now, and we can work on upgrading the changelog tool / tidying this repo's changelog to make them compatible as a separate piece of work?

Reply:
Sounds good to me.